### ISSUE TITLE
Problems with Domino and OmniMIDI 12 & 13.

### ISSUE THREAD

**Issue by Creatinator512** [KaleidonKep99/OmniMIDI#177]
**Problem**
TL;DR: OmniMIDI 12 causes Domino not to work with WinMMWRP and OmniMIDI 13 crashes Domino with the application saying the file was tampered with.

More details:
I got some help from the BMC about this issue, and it seems that Domino has a problem with NTFS file compression, however it should run despite it being compressed. I don't know what is causing the problem, so _please_ help me figure this out.

**Steps to reproduce**
For OmniMIDI 12.x.0.0:
1. Apply WinMM patch to Domino.exe
2. Run Domino.exe. The window shouldn't come up. You'll also see in Task Manager that an error report application will start, but then stop.

For OmniMIDI 13.x.0.0:
1. Run Domino with nothing in its MIDI-OUT ports.
2. Go to Preferences -> MIDI-OUT and set OmniMIDI as Port A's MIDI-OUT.
3. Apply the changes. Immediately you'll see an error message like in the picture.

**Expected behavior**
Domino should run without throwing errors and the soundfont selected in OmniMIDI should work properly.

**Screenshots/Videos**
![image](https://user-images.githubusercontent.com/78432612/106637783-20ae9200-6551-11eb-889c-4728e47cfb5c.png)

**Environment**
 - Operating system: Windows 10, Version 20H2 (Build 19042.746)
 - CPU: Intel(R) Pentium(R) CPU G850 @ 2.90 GHz
 - RAM: 4 GB
 - MIDI application: Domino (Version 1.43 and 1.44)
 - Version of the driver: 12.x.0.0 and 13.x.0.0

**Additional context**
This Domino is the English translation by Hans5958, and because of that, I acknowledge that the actual problem might be a compatibility issue between the two applications.

**Comment by KaleidonKep99** [comment]
I've been checking around for a while, and the message seem to have been spawned by BASS itself.
Something's messing around with the memory, most likely an antivirus.
Try disabling any antivirus that you might have enabled, and try again.

I don't remember if I already told you in the Discord server, but you could also try reinstalling OM13.1 on top of the old install (Safe mode recommended).

Source: https://www.un4seen.com/forum/?topic=14460.msg100510#msg100510

**Comment by Creatinator512** [comment]
I tested if the antivirus was the problem, and the error kept coming up even after I temporarily disabled my antivirus. I will note that it stuck around in the task manager though, and would always jump up to the top when something got opened, regardless of if it was turned off or not.
I'm a bit confused by what you mean by installing OM 13.1 on top of the old install. Can you explain how I would do this, please?

**Comment by KaleidonKep99** [comment]
> 
> 
> I tested if the antivirus was the problem, and the error kept coming up even after I temporarily disabled my antivirus. I will note that it stuck around in the task manager though, and would always jump up to the top when something got opened, regardless of if it was turned off or not.
> I'm a bit confused by what you mean by installing OM 13.1 on top of the old install. Can you explain how I would do this, please?

Just run OM's setup again and restart, and see if that fixes the issue.
It's weird that it's giving a tampering error, since I didn't modify the DLLs.

**Comment by KaleidonKep99** [comment]
I'll close the issue in 2 days for inactivity.

**Comment by Creatinator512** [comment]
I'm very sorry about the inactivity. I gave up on trying to do this since I could just stick with OM 11.2. I thank you very much for doing your best to help me though.

**Comment by Creatinator512** [comment]
If you are still interested to know the result, that error still happens after I ran the OM 13.1 Update Installer and restarting twice replacing OM 11.2. I'm still not sure if I'm doing it correctly, but if you think this isn't worth your time, I understand.

**Comment by KaleidonKep99** [comment]
No no I honestly want to fix this, because if someone has the issue then others probably will too.
Can you post a list of all your installed programs here? (A screenshot from the control panel is fine)

**Comment by Creatinator512** [comment]
![Screenshot 2021-02-15 145339](https://user-images.githubusercontent.com/78432612/107988541-4058a800-6f9e-11eb-982c-14c86f91a288.png)
Some text has been hidden for privacy concerns.

**Comment by KaleidonKep99** [comment]
Please uninstall Norton 360, then try again.

**Comment by Creatinator512** [comment]
Okay. This will likely take some time, but I'll let you know what the result is ASAP.

**Comment by Creatinator512** [comment]
Summary: OM versions 13.2 to 13.5.2 don't fix the issue, and OM 13.5.2 can't load the configurator window. I am still in the process of verifying if Norton 360 is still the problem.

Description: After finding new updates (OM 13.2 to OM 13.5.2), I have determined that they follow the same pattern as the OM 13 versions that were discussed before. OM 13.5.2.0 in particular can't seem to load the configurator entirely, as it will visibly start, but then close according to task manager. I have also found that when the error pops up after changing the MIDI-OUT port to OmniMIDI, I can click back into Domino and continue using it. This doesn't mean that it's still functional, as certain actions like switching tracks or just waiting long enough will cause Domino to not respond. However, before I started experiencing that, it seemed like I could still use it, since looking at the debug log was confirming that it was recognizing note placements (I can't verify if there was actually sound). But after I left the settings menu in Domino, immediately placing a note caused an error to pop up with an "Illegal Operation". This happened again when I switched tracks, but it came up with a log as well. I noted that "basswv.dll" was among one of the files mentioned in the log (Was using OM 13.4.0.0 at that time). After those errors happened, it would throw an error in Domino's notification window that I couldn't read due to mixed up characters, and no sound would play when I placed a note.

**Comment by KaleidonKep99** [comment]
basswv.dll? Could you share the log here?
Also yeah, apparently Visual Studio messed up the configurator's compile... I'll fix it asap.

**Comment by Creatinator512** [comment]
![image](https://user-images.githubusercontent.com/78432612/108890220-4c1a1f00-75db-11eb-92d5-0ce955e8a89c.png)
This happened when switching tracks.

**Comment by Creatinator512** [comment]
OM 13.6.0.0 has fixed every mentioned issue! Thank you for your efforts to combat these bugs!
